noxfile.py

```diff
@@ -148,6 +151,40 @@ def system(session):
         session.run("py.test", "--quiet", system_test_folder_path, *session.posargs)
 
 
+@nox.session(python=CONFORMANCE_TEST_PYTHON_VERSIONS)
+def conformance(session):
```

Comment:
Maybe note that this is just retry conformance and/or rename (to avoid confusion with signed URL conformance tests).